build_stage_repository

```diff
@@ -70,7 +73,7 @@ def create_modulemd(collection, version, stage_dir):
     with open(modulemd_yaml(collection), 'r') as file:
         modules = yaml.safe_load(file)
 
-    modules['data']['artifacts'] = {'rpms': output.splitlines()}
+    modules['data']['artifacts'] = {'rpms': [str(rpm, 'UTF-8') for rpm in output.splitlines()]}
```

Comment:
If you use `universal_newlines=True` when calling `check_output` there is no need to decode (in my experience)